front/src/common/api/osrdEditoastApi.ts

```diff
@@ -34,7 +34,7 @@ const osrdEditoastApi = generatedEditoastApi.enhanceEndpoints({
     },
     deleteV2TrainSchedule: {
       query: (queryArg) => ({
-        url: `/v2/train_schedule/`,
+        url: `/v2/train_schedule`,
         method: 'DELETE',
         body: queryArg.body,
       }),
```

Comment:
Upon further inspection, it seems like `query` is identical to the one in `generatedEditoastApi` so can just be dropped altogether.

Same for `postV2TimetableByIdTrainSchedule`.